docs/book/src/clusterctl/configuration.md

```diff
@@ -160,9 +160,16 @@ all the images in the cert-manager component.
 For situations when resources are limited or the network is slow, the cert-manager wait time to be running can be customized by adding a field to the clusterctl config file, for example:
 
 ```yaml
-  cert-manager-timeout: 15m 
+  cert-manager-timeout: 15m
 ```
 
 The value string is a possibly signed sequence of decimal numbers, each with optional fraction and a unit suffix, such as "300ms", "-1.5h" or "2h45m". Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
 
-If no value is specified or the format is invalid, the default value of 10 minutes will be used.
\ No newline at end of file
+If no value is specified or the format is invalid, the default value of 10 minutes will be used.
+
+
+## clusterctl debugging/logging
```

Comment:
```suggestion
## Debugging/Logging
```

Reply:
thanks again for your review
made the change :)